infrastructure/templates/main.tf.template

```diff
@@ -0,0 +1,21 @@
+terraform {
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+
+   backend "s3" {
+    bucket = "S3_BUCKET_TERRAFORM"
+    key    = "terraform.tfstate"
+    region = "REGION"
```

Comment:
you can set the region from outside
[config](https://developer.hashicorp.com/terraform/language/settings/backends/configuration)

`terraform init -input=false  -backend-config="bucket=irene-bucket"`